lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -33,15 +33,15 @@ def build_sphinx_html(source_dir, doctree_dir, html_dir, extra_args=None):
 
 
 def test_tinypages(tmp_path):
-    shutil.copytree(Path(__file__).parent / 'tinypages', tmp_path,
+    shutil.copytree(Path(__file__).parent / 'data/tinypages', tmp_path,
```

Comment:
While all these lines are being changed, it might be worth adding "tinypages_data_path = Path(__file__).parent / "data" / "tinypages"` variable to the file and re-using it?

Reply:
Sure, done.